pkg/kubelet/server/server.go

```diff
@@ -285,6 +285,7 @@ const pprofBasePath = "/debug/pprof/"
 
 // InstallDeguggingHandlers registers the HTTP request patterns that serve logs or run commands/containers
 func (s *Server) InstallDebuggingHandlers(criHandler http.Handler) {
+	glog.Infof("Adding Debug Handlers")
```

Comment:
These are written directly to the kubelet logs, but sure I can make it a bit more clear.